src/code/lb_rtc.c

```diff
@@ -97,7 +92,7 @@ void lbRTC_SetTime(lbRTC_time_c* time) {
     if (common_data.time.rtcEnabled == 1 && !common_data.time.rtcCrashed) {
         time->weekday = lbRTC_Week(time->year, time->month, time->day);
         mq = padmgr_LockSerialMesgQ();
-        func_800FE670_jp(mq, time);
+        lbrtc_setTime(mq, (OSRTCTime*)time);
```

Comment:
Looks like `lbRTC_time_c` and `OSRTCTime` are the same struct. Could you merge them?

Reply:
The existence of these two structs are known by the flashrom structs in the symbols, should I just have `lbRTC_time_c` have an `OSRTCTime` member?

Reply:
Could you point me to those symbols? I want to take a look